### ISSUE TITLE
Support NATS connector plugin and NATS as event store

### ISSUE THREAD

**Issue by udesh** [WeBankFinTech/EventMesh#250]
Is there support to use [NATS](https://docs.nats.io/) as the event store.

**Comment by xwm1992** [comment]
ok, i got it, we will have a discuss for this feature.

**Comment by udesh** [comment]
Yes sure, I would like to join and contribute if possible for supporting this.

**Comment by xwm1992** [comment]
ok, if you have any problem, let's communicate with dev@eventmesh.apache.org email address.

**Comment by qqeasonchen** [comment]
@udesh you can join our wechat group and let me know who you are, or you can send email to dev@eventmesh.apache.org，welcome to contirbute anyway.
